### PR TITLE
Expose like and popularity on the /score API endpoint.

### LINKED ISSUE #3996: Expose all three score components
In our new design, the package score has three components:

1. Points (aka Pub Points)
1. Popularity
1. Likes

We currently expose these via several APIs:

For likes: https://pub.dev/api/packages/pana/likes
For popularity: https://pub.dev/api/packages/pana/metrics?pretty
For points https://pub.dev/api/packages/pana/metrics?pretty & https://pub.dev/api/packages/pana/score

I suggest we expand the `metrics` and `score` APIs to have all three score components:

```
{
  "grantedPoints":60,
  "maxPoints":110,
  "popularityScore": 0.6633297696569151,
  "likes": 7,
  "lastUpdated":"2020-09-03T22:11:22.940284Z"
}
```

### PR DESCRIPTION
**Pull request by isoos**
Fixes #3996.

### CODE REVIEW

File: pkg/client_data/lib/package_api.dart [hunk 1]
```diff
@@ -154,11 +154,15 @@ class VersionInfo {
 class VersionScore {
   final int grantedPoints;
   final int maxPoints;
+  final int likeCount;
```
Comment (jonasfj) [inline pkg/client_data/lib/package_api.dart:157]: Can we rename this to `likes`

File: pkg/client_data/lib/package_api.dart [hunk 2]
```diff
@@ -154,11 +154,15 @@ class VersionInfo {
...
+  final double popularityScore;
```
Comment (jonasfj) [inline pkg/client_data/lib/package_api.dart:158]: Can we rename this `popularity`

**Review by jonasfj — APPROVED** [review]

File: pkg/client_data/lib/package_api.dart (hunk 1 above)
Comment (isoos) [reply pkg/client_data/lib/package_api.dart:157]: I'd argue that we should use `likeCount` as opposed to `likes` not only here, but other places where we've had `likes` before. `List<Like>` should be `likes` (and `likes` should be `List<Like>`), while the cardinality of the same list is the `likeCount`.

File: pkg/client_data/lib/package_api.dart (hunk 2 above)
Comment (isoos) [reply pkg/client_data/lib/package_api.dart:158]: I was pondering on this, and thought that if we'd call it `popularity`, then we should also transform it into the `0-100` range, and switch the type to `int`. wdyt?

Comment (jonasfj) [reply pkg/client_data/lib/package_api.dart:158]: I'm also okay keeping it this way. let's not over think it, long term I think we want absolute numbers here.

File: pkg/client_data/lib/package_api.dart (hunk 1 above)
Comment (jonasfj) [reply pkg/client_data/lib/package_api.dart:157]: Seems fine either way, I just wanted to try to keep the external API neat.

**Review by jonasfj — APPROVED** [review]

**Pull request merged**